draft-ietf-rats-architecture.md

```diff
@@ -354,72 +354,65 @@ while other slots cannot. But other slots can communicate with the main
-each router is an Attester and the main router is the Lead Attester.
+and therefore they are managed and appraised via this main router's help.
+So, in this case, the multi-chassis router is the Composite Device,
+each router is an Attester and the main router is the transiting Attester.
 
 {{composite}} depicts the conceptual data flow for a Composite Device.
 
 ~~~~
-                   .-----------------------------.
-                   |           Verifier          |
-                   '-----------------------------'
-                                   ^
-                                   |
-                                   | Composite
-                                   | Evidence
-                                   |
-.----------------------------------|-------------------------------.
-| .--------------------------------|-----.      .------------.     |
-| |                      .------------.  |      |            |     |
-| |                      |  Attesting |<--------| Attester B |-.   |
-| |                      |Environment |  |      '------------. |   |
-| |  .----------------.  |            |<----------| Attester C |-. |
-| |  |     Target     |  |            |  |        '------------' | |
-| |  | Environment(s) |  |            |<------------| ...        | |
-| |  |                |  '------------'  | Evidence '------------' |
-| |  |                |            ^     |    of                   |
-| |  |                |------------/     | Attesters               |
-| |  '----------------'  Collecting      | (via Internal Links or  |
-| |                      Claims          | Network Connections)    |
-| |                                      |                         |
-| | Lead Attester A                      |                         |
-| '--------------------------------------'                         |
-|                                                                  |
-|                       Composite Device                           |
-'------------------------------------------------------------------'
+              .-----------------------------.
+              |           Verifier          |
+              '-----------------------------'
+                             ^
+                             | Evidence of
+                             | Composite Device
+                             |
+.----------------------------|----------------------------------.
+| .--------------------------|--------.      .------------.     |
+| |                .----------------. |<-----| Attester B |     |
+| |  Collect       | Attesting      | |      '------------'-.   |
+| |  Claims .----->| Environment(s) | |<-------| Attester C |   |
+| |         |      '----------------' |        '------------'-. |
+| | .----------------.                |<---------| ...        | |
+| | | Target         |                | Evidence '------------' |
+| | | Environment(s) |                | of Attesters            |
```

Comment:
Should say "Evidence of composite Attesters" (?)

Reply:
@nedsmith In my opinion, no